internal/metrics/remote_write.go

```diff
@@ -384,6 +431,21 @@ func (r *RemoteWrite) writeRequest(series []Series, client WriteClient) bool {
 	return true
 }
 
+func (r *RemoteWrite) removeServerCaFiles() {
+	fileInfo, err := ioutil.ReadDir(r.dataDir)
+	if err != nil {
+		log.Errorf("could not read %s", r.dataDir)
```

Comment:
s/could not/cannot/ for consistency

Reply:
done

Reply:
It seems the edit is not there. Can you please double check?